### PR TITLE
New Pull Request

### PR DESCRIPTION
**Pull request by BensonCasey7**

### CODE REVIEW

**Review by floatkeera — APPROVED** [review]
good

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ wthr. utilizes the OpenWeatherMap API for real-time weather information, as well
 This test app can be viewed at https://floatwthr.firebaseapp.com/
 
 ## Dependencies
-- npmf
```
Comment (floatkeera) [inline README.md:8]: not good

**Review by floatkeera — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BensonCasey7) [reply README.md:8]: @floatkeera test

**Review by floatkeera — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (floatkeera) [reply README.md:8]: @BensonCasey7  hello

**Review by floatkeera — DISMISSED** [review]

**Review by floatkeera — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (BensonCasey7) [reply README.md:8]: tets

**Comment by BensonCasey7** [comment]
testing

**Comment by floatkeera** [comment]
asdfasdf

**Comment by BensonCasey7** [comment]
@floatkeera

File: README.md (hunk 1 above)
Comment (floatkeera) [reply README.md:8]: lol

**Review by floatkeera — APPROVED** [review]

**Pull request merged**